### PR TITLE
add sanitization yotpo settings

### PR DESCRIPTION
**Pull request by woodshedsomali**
https://yotpoent.atlassian.net/browse/SPK-1784

### CODE REVIEW

**Review by p-bartosz — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ svn add * --force
 
 7. run 'svn diff' to verify the code changes before you upload the code
 6. Push the changes:
-svn ci -m 'Version `<<new version number>>`' --username Yotpo --password yotpo2010
+svn ci -m 'Version `<<new version number>>`' --username Yotpo --password Y0!po2024
```
Comment (vladimir-marinov-yotpo) [inline README.md:33]: This is a public repo. Is this okay?

**Review by vladimir-marinov-yotpo — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (woodshedsomali) [reply README.md:33]: Great catch! I don't it is okay to keep SVN pass publicly available, but it was like this so far. I will remove the password here and add confluence page link instead. The question is whether this repo should stay public?

Comment (vladimir-marinov-yotpo) [reply README.md]: I think it is fine to leave this repo public. The plug-in is going to be installed on woocommerce shop and they can look at the code. Also I think this repo needs to be public so the WooCommerce repository can get the new versions.

Comment (woodshedsomali) [reply README.md]: Code of the plugin itself is publicly available here
https://plugins.svn.wordpress.org/yotpo-social-reviews-for-woocommerce/ 
and for sometime here: 
https://github.com/wp-plugins/yotpo-social-reviews-for-woocommerce

As far as I understand, this repo is used only by Yotpo to keep track on changes what is on plugins.svn.wordpress. 

@vladimir-marinov-yotpo Could you please approve the PR?

Comment (vladimir-marinov-yotpo) [reply README.md]: Sure.

**Pull request merged**